3-enrich/stream-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/kinesis/KinesisEnrichApp.scala

```diff
@@ -344,6 +347,15 @@ class KinesisEnrichConfig(config: Config) {
   private val kafka = enrich.getConfig("kafka")
   val kafkaBrokers = kafka.getString("brokers")
 
+  private val nsq = enrich.getConfig("NSQ")
+  val nsqGoodSourceTopicName = nsq.getString("good-source-topic")
+  val nsqGoodSourceChannelName = nsq.getString("good-source-channel")
+  val nsqGoodSinkTopicName = nsq.getString("good-sink")
+  val nsqBadTopicName = nsq.getString("bad-sink")
+  val nsqHost = nsq.getString("nsq-host")
+  val nsqdPort = nsq.getInt("nsqd-port")
+  val nsqlookupdPort = nsq.getInt("nsqlookupd-port")
```

Comment:
Now that I think about it, it's pointless to repeat nsq, since the globbing config is already called, let's refactor it as:

```config
NSQ {
  host = ...
  port = ...
  lookup-port = ...
}
```